### ISSUE TITLE
Add scores of particular test 

### ISSUE THREAD

**Issue by Abhinavcode13** [avinashkranjan/Friday#189]
please assign this issue to me @avinashkranjan

**Comment by Apreksha** [comment]
Hello @avinashkranjan. I'm a GSSoCC'23 contributor. I would like to add the scores of particular test. Could you please assign this task to me.

**Comment by avinashkranjan** [comment]
Go Ahead @Abhinavcode13

**Comment by ETHER-BHASHI** [comment]
I would like to add the scores of particular test. May you please assign this task to me.

**Comment by avinashkranjan** [comment]
Any Update on this @Apreksha, or Should I assign this to @ETHER-BHASHI

**Comment by Apreksha** [comment]
Hi @avinashkranjan, this issue hasn't been assigned to me.

**Comment by avinashkranjan** [comment]
Oh Sorry, Hey @Abhinavcode13 any updates on this?

**Comment by Abhinavcode13** [comment]
yeah you can assign someone else too , im too working on it @avinashkranjan

**Comment by avinashkranjan** [comment]
@Apreksha @ETHER-BHASHI If you want collaborate and work together you can, Assigning this to all three of you.
